.travis.yml

```diff
@@ -1,5 +1,4 @@
 sudo: false
-cache: bundler
```

Comment:
I thought this might get travis to bundle without the `--deployment` flag, which is what broke it on my first attempt.

Do you mind if I add the appraisal gem in order to maintain Gemfiles?  That would be the easiest way to generate separate gemfiles without travis needing to do anything special